server/views/macros/prisonerSearchOffenceCodeDecisionInput.njk

```diff
@@ -0,0 +1,46 @@
+{% from "govuk/components/input/macro.njk" import govukInput %}
```

Comment:
Is this (and the prisoner version) a copy of the njk(s) made by Nat for the incident role prisoner search?

Reply:
..maybe not, as incident role has different variable names, but maybe